Source/TextToSpeech-Component/Runtime/Provider/FileTextToSpeechProvider.cs

```diff
@@ -70,37 +70,42 @@ protected virtual string GetPathToFile(string filename)
         /// </summary>
         /// <param name="filePath">Relative path where the cached file is stored.</param>
         /// <returns>A byte array containing the contents of the file.</returns>
-        protected virtual byte[] GetCachedFile(string filePath)
+        protected virtual async Task<byte[]> GetCachedFile(string filePath)
         {
             if (Application.isPlaying)
             {
-                return FileManager.Read(filePath);
+                return await FileManager.Read(filePath);
             }
             else
             {
-                return File.ReadAllBytes(Path.Combine(Application.streamingAssetsPath, filePath));
+                return await File.ReadAllBytesAsync(Path.Combine(Application.streamingAssetsPath, filePath));
```

Comment:
It does not compile for me with ReadAllBytesAsync. The previous line still works fine, though.

Reply:
not sure, why my Rider was suggesting this and it compiled in Unity. I reverted that line.